core/store/src/trie/mem/parallel_loader.rs

```diff
@@ -214,6 +214,7 @@ impl ParallelMemTrieLoader {
         arena: &mut impl ArenaMut,
     ) -> Result<MemTrieNodeId, StorageError> {
         // Figure out which range corresponds to the prefix of this subtree.
+        // TODO(reshardingV3) This seems fragile, potentially does not work with mapping.
```

Comment:
This is only used for flat store, can remove this comment.

Reply:
Let me remove it later, I would like to understand this part of code more, I had some concerns before.